Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -741,12 +734,12 @@ - (void)_setGroupIdentifier:(NSString *)groupIdentifier
 
 - (BOOL)_respectsImageOrientation
 {
-    return _respectsImageOrientation;
+    return _pageConfiguration->preferences()->shouldRespectImageOrientation();
```

Comment:
I think preferences can be null, and this would introduce a crash.  In WKWebView._setupPageConfiguration we have already ensured that the preferences is not null, but that is not the case here.

Reply:
I assumed it was ok because of the other unchecked access below, but in fact that access through `self.preferences` goes through a special getter that allocates a WebPreferences object if not there yet!
Ok, I'll see what I can do... I'd prefer not to have a redundant value in both PageConfiguration and WebPreferences -- but please let me know if you disagree.